From b9d5eab53ab07340c14d0e80ea00b8024395594b Mon Sep 17 00:00:00 2001
From: Mihai Moldovan <ionic@ionic.de>
Date: Tue, 29 Sep 2020 18:30:16 +0200
Subject: [PATCH] Use static libcheck version.

Within Debian, only the static libcheck library is shipped. Use it correctly,
since the static library requires additional private libraries and definitions.
---
 tests/hawkey/CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/tests/hawkey/CMakeLists.txt b/tests/hawkey/CMakeLists.txt
index 29402d63..601bb4bc 100644
--- a/tests/hawkey/CMakeLists.txt
+++ b/tests/hawkey/CMakeLists.txt
@@ -29,6 +29,8 @@ set_target_properties(test_hawkey_main PROPERTIES COMPILE_FLAGS -fPIC)
 target_link_libraries(test_hawkey_main
     libdnf
     ${CHECK_LDFLAGS}
+    ${CHECK_STATIC_LIBRARIES}
+    ${CHECK_STATIC_LDFLAGS}
     ${SOLV_LIBRARY}
     ${SOLVEXT_LIBRARY}
     ${RPMDB_LIBRARY}
-- 
2.26.2

